Makefile

```diff
@@ -64,6 +66,8 @@ install_client:
           _build/src/tools/typed_ssl.mli \
           _build/src/tools/typed_ssl.cmi \
           _build/src/tools/unix_fd.cmi \
+	  _build/src/tools/arakoon_etcd.cmi \
+	  _build/src/tools/arakoon_url.cmi \
```

Comment:
`Each line in the recipe must start with a tab` from
https://www.gnu.org/software/make/manual/html_node/Recipe-Syntax.html